### PR TITLE
COMMERCE-980 Only add back arrow for accounts if they have more than 1

### PR DESCRIPTION
**Pull request by alecsloan**
Let me know if you have any questions.

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Comment by iovaneale** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:stop

**Comment by ethanbustad** [comment]
ci:test

**Comment by marco-leo** [comment]
@alecsloan please rebase this

**Comment by alecsloan** [comment]
ci:test

File: commerce-frontend-taglib/src/main/resources/META-INF/resources/account_selector/AccountSelector.es.js [hunk 1]
```diff
@@ -244,4 +246,4 @@ AccountSelector.STATE = {
 };
 
 export {AccountSelector};
-export default AccountSelector;
\ No newline at end of file
+export default AccountSelector;
```
Comment (ethanbustad) [inline commerce-frontend-taglib/src/main/resources/META-INF/resources/account_selector/AccountSelector.es.js]: don't think you meant to remove the trailing newline here

**Review by ethanbustad — APPROVED** [review]

**Comment by alecsloan** [comment]
ci:test

**Pull request merged**